README.md

```diff
@@ -157,6 +174,10 @@ rarefaction depth of 1500**
 **Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs**  
 `sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
 
+**Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs. Write the full output.**  
+`sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
```

Comment:
Also, can you make this ``-o example7``? This makes it easier to run the commands back-to-back (which I do as part of testing, but which we'll ultimately want to automate in the future).